backend/core/src/user/user.controller.ts

```diff
@@ -48,12 +49,19 @@ export class UserController {
   @Post()
   @UseGuards(OptionalAuthGuard, AuthzGuard)
   @ApiOperation({ summary: "Create user", operationId: "create" })
-  async create(@Body() dto: UserCreateDto): Promise<UserDtoWithAccessToken> {
+  async create(@Body() dto: UserCreateDto): Promise<UserDto> {
     const user = await this.userService.createUser(dto)
-    const accessToken = this.authService.generateAccessToken(user)
     // noinspection ES6MissingAwait
-    void this.emailService.welcome(user)
-    return mapTo(UserDtoWithAccessToken, { ...user, accessToken })
+    void this.emailService.welcome(user, dto.appUrl)
```

Comment:
I know this is not the case now in the codebase, but in the future I would like to move all the business logic inside services. So for example that emailService.welcome  would go inside userService. It's just something I'll try to clean up soon up and mentioning it here to spread the info.